system/settings/settings.xml

```diff
@@ -2544,7 +2548,7 @@
           <level>2</level>
           <default>false</default>
           <dependencies>
-            <dependency type="visible">
+            <dependency type="enable">
```

Comment:
showing DTSHD on a system that does not support it is the same nonsense as showing VAAPI on a NVIdia system. Settings are not a means for educating the clueless users.